packages/frontend/src/features/CohortBuilder/CohortPanel.tsx

```diff
@@ -289,20 +290,30 @@ export const CohortPanel = ({
               fields={fields}
               filter={cohortFilters}
             />
-
-            <CountsValue
-              label={guppyConfig.nodeCountTitle}
+            <div className="flex justify-between flex-row items-center mb-2">
+              <Gen3Button
```

Comment:
Would it be an easy win to make this button an `Accordion` like in Profile? That would separate it from the fake `CountsValue` button and also allow it to take up the entire horizontal visual space to match the table below.

Reply:
Won't do for now